pom.xml

```diff
@@ -23,7 +23,7 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.version>2.16.5.Final</quarkus.version>
+        <quarkus.version>3.0.0.CR1</quarkus.version>
 
         <quarkus.poi.version>1.0.4</quarkus.poi.version>
```

Comment:
Please use [2.0.0.Alpha2](https://github.com/quarkiverse/quarkus-poi/releases/tag/2.0.0.Alpha2) before merging.

Reply:
Thansk for the tip! and is it `<quarkus.poi.version>2.0.0.Alphat</quarkus.poi.version>`?

Reply:
Yes, this is the latest released one, having Quarkus 3.0.0.Alpha5 as dependency.